src/daemon/org.projectatomic.rpmostree1.xml

```diff
@@ -92,6 +92,9 @@
 
          Available options:
          "reboot" (type 'b')
+         "install-packages" (type 'as')
+         "uninstall-packages" (type 'as')
+         "install-local-packages" (type 'ah')
```

Comment:
I think the assumption here is that missing keys imply carrying over, which nicely matches the current behaviour anyway on the daemon side.

As for the `yum shell`, yeah, we can add a e.g. `rpm-ostree txn` type of thing, though we probably still want that functionality to be accessible from the usual commands as well.